Buildings/Experimental/DHC/EnergyTransferStations/Cooling/Examples/DirectControlled.mo

```diff
@@ -0,0 +1,167 @@
+within Buildings.Experimental.DHC.EnergyTransferStations.Cooling.Examples;
+model DirectControlled "Example model for direct cooling energy transfer station 
+  with in-building pumping and contrlled district return temperature"
+  extends Modelica.Icons.Example;
+  package Medium=Buildings.Media.Water
+    "Water medium";
+  parameter Modelica.Units.SI.MassFlowRate mDis_flow_nominal=0.5
+    "Nominal mass flow rate of district cooling supply"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.HeatFlowRate Q_flow_nominal=-18000
+    "Nominal cooling load (Negative for cooling)"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mBui_flow_nominal=-Q_flow_nominal/(cp*(18-7))
+    "Nominal mass flow rate of building cooling supply"
+    annotation(Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.SpecificHeatCapacity cp=Medium.specificHeatCapacityCp(
+    Medium.setState_pTX(
+      Medium.p_default,
+      Medium.T_default,
+      Medium.X_default))
+    "Default specific heat capacity of medium";
+  inner Modelica.Fluid.System system
+    "System properties and default values"
```

Comment:
Is `inner Modelica.Fluid.System` used in MBL models ? Please check if MBL uses any other approach

Reply:
Yes, if you look at `Buildings.Fluid.MixingVolumes.Validation.MixingVolume`, it uses this `inner Modelica.Fluid.System`.
![image](https://user-images.githubusercontent.com/82961137/159552535-ac5743c9-cadc-4fd5-8193-337519f0197c.png)

Reply:
Great! thanks for pointing it out.  
Most of the MSL components need the `inner Modelica.Fluid.System ` to be added in the model to run simulations without error. I assumed MBL components may not need it. Good to know that MBL also uses it!